src/libsystemd/sd-bus/test-bus-watch-bind.c

```diff
@@ -212,20 +210,20 @@ int main(int argc, char *argv[]) {
 
         /* We use /dev/shm here rather than /tmp, since some weird distros might set up /tmp as some weird fs that
          * doesn't support inotify properly. */
-        assert_se(mkdtemp_malloc("/dev/shm/systemd-watch-bind-XXXXXX", &d) >= 0);
+        ASSERT_OK(mkdtemp_malloc("/dev/shm/systemd-watch-bind-XXXXXX", &d));
 
         path = strjoina(d, "/this/is/a/socket");
 
-        assert_se(pthread_create(&server, NULL, thread_server, path) == 0);
-        assert_se(pthread_create(&client1, NULL, thread_client1, path) == 0);
-        assert_se(pthread_create(&client2, NULL, thread_client2, path) == 0);
+        ASSERT_OK_EQ_ERRNO(pthread_create(&server, NULL, thread_server, path), 0);
```

Comment:
ASSERT_OK_EQ_ERRNO is incorrect for pthread_create().  pthread_create() returns 0 on success or a positive error number on failure (never negative), and does not set errno. So this macro will never trigger the <0 check since pthread_create() returns positive errors, and may report a meaningless errno value if pthread_create() fails